hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/TestAWSV4AuthValidator.java

```diff
@@ -64,15 +67,25 @@ public static Collection<Object[]> data() {
                 "577efef23edd43b7e1a59",
             "5d672d79c15b13162d9279b0855cfba" +
                 "6789a8edb4c82c400e06b5924a6f2b5d7",
-            "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY"
+            "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY",
+            true
+        },
+        {
+            "PUT\n" +
```

Comment:
Can we add a few more test cases? Will the regex flag `AWS4-ZAVC-HJUA123` in the start of the header? The regular expression added is the main entity to check here.